rabbitmq-message-broker.html

```diff
@@ -43,6 +43,14 @@ <h1 class="section-heading">Client Requirements</h1>
             <li>It SHALL ensure that published event types defined in edition-toulouse of the <a href="https://github.com/eiffel-community/eiffel">Eiffel protocol</a> comply with protocol definitions.</li>
             <li>It MAY publish messages on Eiffel-like syntax, containing event types not defined in the <a href="https://github.com/eiffel-community/eiffel">Eiffel protocol</a>, as well as other other types of messages completely unrelated to the Eiffel protocol.</li>
             <li>It SHALL support routing keys.</li>
+            <li>
+              It SHOULD use a routing key on the form <code>eiffel.&lt;type&gt;.&lt;tag&gt;.&lt;domainid&gt;</code>, where:
+              <ul>
+                <li><code>type</code> is the type of the published Eiffel event (i.e. its <code>meta.type</code> member), e.g. EiffelArtifactCreatedEvent.</li>
+                <li><code>tag</code> is an implementation-specific tag. It can be any non-empty string but must not contain a period.</li>
+                <li><code>domainid</code> is the non-empty string representing the domain the event applies to. It corresponds to the <code>meta.source.domainId</code> member of an Eiffel event.</li>
```

Comment:
Should we restrict the domain to not contain any periods? I don't know how the domain id is currently used so I don't know if such a restriction would be a problem, but allowing it feels icky.